config/initializers/dynflow.rb

```diff
@@ -1,2 +1,2 @@
 # Ignore Dynflow tables when schema-dumping. Dynflow tables are handled automatically by Dynflow.
-ActiveRecord::SchemaDumper.ignore_tables << /^dynflow_.*$/
+ActiveRecord::SchemaDumper.ignore_tables << '^dynflow_*'
```

Comment:
I'm not 100% sure about this, the docs seem to indicate that '^dynflow_*$' should work, but it didn't seem to and allowed everything through.  '^dynflow_*' appeared to work fine:
```
$ grep dynflow_ db/structure.sql 
    dynflow_execution_plan_id character varying,
    dynflow_step_id integer,
```